test/e2e/pageobjects/OIEOktaLogin.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable max-len */
+class OIEOktaLogin {
+  get signinForm() { return $('form[data-se="o-form"]');}
+  get signinUsername() { return  $('[name="identifier"]'); }
+  get signinPassword() { return $('[name="credentials.passcode"]'); }
+  get signinSubmitBtn() { return $('[data-type="save"]'); }
+
+  async signin(username, password) {
+    await this.waitForLoad();
+    await this.signinUsername.then(el => el.setValue(username));
+    await this.signinPassword.then(el => el.setValue(password));
+    await this.signinSubmitBtn.then(el => el.click());
+
```

Comment:
For some reason, when org is in "identifier first" flow, this code block didn't work correctly.
Logically it should work but for some reason, it didn't. Will follow up in a subsequent PR